x-pack/plugins/security_solution/public/common/components/events_tab/events_query_tab_body.tsx

```diff
@@ -86,6 +99,52 @@ const EventsQueryTabBodyComponent: React.FC<EventsQueryTabBodyComponentProps> =
   const { globalFullScreen } = useGlobalFullScreen();
   const ACTION_BUTTON_COUNT = 5;
   const tGridEnabled = useIsExperimentalFeatureEnabled('tGridEnabled');
+  const [defaultNumberFormat] = useUiSetting$<string>(DEFAULT_NUMBER_FORMAT);
+  const [showExternalAlerts, setShowExternalAlerts] = useState<boolean>(false);
+  const leadingControlColumns = useMemo(() => getDefaultControlColumn(ACTION_BUTTON_COUNT), []);
+
+  const toggleExternalAlerts = useCallback(() => setShowExternalAlerts((s) => !s), []);
+
+  const getSubtitle = useCallback(
+    (totalCount: number) =>
+      `${SHOWING}: ${numeral(totalCount).format(defaultNumberFormat)} ${UNIT(totalCount)}`,
+    [defaultNumberFormat]
+  );
+
+  const histogramExtraProps = useMemo(
+    () => ({
+      ...(showExternalAlerts
+        ? {
+            ...alertsHistogramConfig,
+            id: ALERT_HISTOGRAM_ID,
+            defaultModel: alertsDefaultModel,
+            pageFilters: externalAlertPageFilters,
```

Comment:
Probably we should do the same as in line 139

Reply:
the historgram actually does not need this prop, we give it a histogramType, that I believe gets resolved on the server. The old code didn't actually have that passed to the histogram :D.